### PR TITLE
[UXE-4708] feat: allow accounts access from a list to console

### PR DESCRIPTION
**Pull request by lucasmendes21**
### What is the new behavior introduced by this PR?
allow accounts access from a list to console

### Does this PR introduce breaking changes?
- [ ] No
- [x] Yes 

### Does it have a link on Figma?
no

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/services/account-services/list-client-ids-released-for-console-service.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+
+export const listClientIdsReleasedForConsoleService = async () => {
```
Comment (peterpaulo-azion) [inline src/services/account-services/list-client-ids-released-for-console-service.js:3]: Esse serviço aqui já possui testes?

**Review by HerbertJulio — APPROVED** [review]

File: src/services/account-services/list-client-ids-released-for-console-service.js (hunk 1 above)
Comment (HerbertJulio) [reply src/services/account-services/list-client-ids-released-for-console-service.js:4]: ☝️

File: src/services/account-services/list-client-ids-released-for-console-service.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const adapt = (httpResponse) => {
+  const environment = getEnvironment()
+
+  if (environment === 'production') {
+    return httpResponse?.body?.production?.client_ids ?? []
+  }
+  return httpResponse?.body?.stage?.client_ids ?? []
+}
```
Comment (peterpaulo-azion) [inline src/services/account-services/list-client-ids-released-for-console-service.js:19]: Sugestão de melhoria na legibilidade.

```suggestion
const adapt = (httpResponse) => {
  const { body } = httpResponse || {};
  const { production, stage } = body || {};
  const environment = getEnvironment();

  const clientIdsPath = environment === 'production' ? production : stage;
  return clientIdsPath?.client_ids ?? [];
}
```

**Review by peterpaulo-azion — DISMISSED** [review]

File: azion.config.cjs [hunk 3]
```diff
@@ -242,6 +242,19 @@ const backRules = [
...
     }
+  },
+  {
+    name: 'Route List Client Ids to Github Pages',
+    description: 'this route will get the client ids of the accounts released to the console',
+    match: '^/api/allowed-accounts',
+    behavior: {
+      forwardCookies: true,
+      setOrigin: {
+        name: 'origin-github-pages-allowed-accounts',
```
Comment (vinigfer) [inline azion.config.cjs:253]: We are not actually using github pages
```suggestion
    name: 'Route List Client Ids to Github',
    description: 'This route will get the client ids of the accounts allowed to use the console',
    match: '^/api/allowed-accounts',
    behavior: {
      forwardCookies: true,
      setOrigin: {
        name: 'origin-github-allowed-accounts',
```

File: azion.config.cjs [hunk 4]
```diff
@@ -281,7 +294,13 @@ const AzionConfig = {
       type: 'single_origin',
       hostHeader: `api.azion.com`,
       addresses: [`api.azion.com`]
-    }
+    },
+    {
+      name: 'origin-github-pages-allowed-accounts',
```
Comment (vinigfer) [inline azion.config.cjs:299]:
```suggestion
      name: 'origin-github-allowed-accounts',
```

**Review by vinigfer — APPROVED** [review]

**Pull request merged**